test/augmentation/apply/test_tf_applier.py

```diff
@@ -158,6 +158,7 @@ def test_tf_applier_pandas(self):
         applier = PandasTFApplier([square], policy)
         df_augmented = applier.apply(df, progress_bar=False)
         df_expected = pd.DataFrame(dict(num=[1, 16, 81]), index=[0, 1, 2])
+        self.assertEqual(df_augmented.num.dtype, "int64")
```

Comment:
Did we check that these fail before updating PandasTFApplier?

Reply:
Just checked, I needed to add some strings to the df to make it fail. If it it all ints, then the dtype stays as int64.